src/content/tools/devtools/release-notes/release-notes-2.37.2-src.md

```diff
@@ -1,6 +1,6 @@
-# DevTools 2.37.1 release notes
+# DevTools 2.37.2 release notes
```

Comment:
I just want to doublecheck. This PR mostly looks great, but shouldn't this file be renamed to release-notes-2.37.2-src.md? (It's still ...2.37.1-src.md).

Reply:
Wait, I take it back. It has the correct name now.